packages/pwa-kit-dev/src/utils/resolver-utils.js

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import path from 'path'
+import {getConfig} from '@salesforce/pwa-kit-runtime/utils/ssr-config'
+
+const EXTENSION_NAMESPACE = '@salesforce'
+const EXTENSION_PREFIX = 'extension'
+const NODE_MODULES = 'node_modules'
+const SDK_COMPONENT_MAP = {
+    'app/routes': '/ssr/universal/components/routes'
+}
```

Comment:
are you planning to refactor/replace the sdkReplacementPlugin? https://github.com/SalesforceCommerceCloud/pwa-kit/blob/3b8566482bbb22d9991b7f052f454bee2828270d/packages/pwa-kit-dev/src/configs/webpack/plugins.js#L66

Reply:
So the interesting part of multi-extensibility and how I implemented it, e.g. using the pwa-kit-react-sdk as the final destination to find the module you are trying to import, is that technically we can replace the sdkReplacementPlugin by updating this component map. 

So you are asking the right questions here. I guess now, I ask, what do you think? I feel like it allows us to simplify the overall landscape of the webpack configuration.

Reply:
I think you made a good decision! 👍